src/metricCollector.ts

```diff
@@ -59,14 +59,16 @@ export class MetricCollector {
       }
       this.logger.info('added queue', name);
       this.queuesByName.set(name, {
-        name,
-        queue: new Queue(name, {
-          ...this.bullOpts,
-          connection: this.defaultRedisClient,
-        }),
-        prefix: this.bullOpts.prefix || 'bull',
-        queueEvents: new QueueEvents(name)
-      });
+				name,
+				queue: new Queue(name, {
+					...this.bullOpts,
+					connection: this.defaultRedisClient,
+				}),
+				prefix: this.bullOpts.prefix || 'bull',
+				queueEvents: new QueueEvents(name, {
```

Comment:
we should check if `...this.bullOpts` is relevant here as well..

Reply:
I believe "prefix" option is passed thru these bullOpts, which seems required for QueueEvents to look at the right redis keys
https://api.docs.bullmq.io/interfaces/QueueEventsOptions.html#prefix